editors/vscode/client/extension.ts

```diff
@@ -297,52 +287,28 @@ export async function activate(context: ExtensionContext) {
   updateStatsBar(configService.config.enable);
 
   if (configService.config.enable) {
-    await startClient(client);
-  }
-}
-
-// Starts the client, it does not check if it is already started
-async function startClient(client: LanguageClient | undefined) {
-  if (client === undefined) {
-    return;
+    await client.start();
   }
-  await client.start();
-  // ToDo: refactor it on the server side.
-  // Do not touch watchers on client side, just simplify the start of the server.
-  const configFiles = await findOxlintrcConfigFiles();
-  await sendDidChangeWatchedFilesNotificationWith(client, configFiles);
 }
 
-export function deactivate(): Thenable<void> | undefined {
+export async function deactivate(): Promise<void> {
   if (!client) {
     return undefined;
   }
-  return client.stop();
+  await client.stop();
+  client = undefined;
 }
 
+// FileSystemWatcher are not ready on the start and can take some seconds on bigger repositories
```

Comment:
That is the relevant part of the gif

Reply:
Looks like this is a "me" problem: https://github.com/microsoft/vscode/issues/245211